interp/dumpglob.mli

```diff
@@ -19,11 +19,12 @@ type glob_output =
   | MultFiles                   (* one glob file per .v file *)
   | File of string              (* Single file for all coqc arguments *)
 
-(* Default "NoGlob" *)
-val set_glob_output : glob_output -> unit
+val push_output : glob_output -> unit
+val pop_output : unit -> unit
 
 val pause : unit -> unit
 val continue : unit -> unit
+[@@ocaml.deprecated "Use pop_output"]
```

Comment:
Should be "use pop_output NoGlob"

Reply:
I don't think so, `pop_output` does not take an argument. But this does show that the names are indeed not optimal. I'm currently most in favor of `override_output` and `restore_output`. That seems to capture semantics pretty well. Would that be something everybody could agree on?